src/mixins/resizeConfig.js

```diff
@@ -486,7 +482,7 @@ export default {
     },
     setPointAttributes(point, cursorDirection) {
       point.attr({
-        root: { display: 'none' },
+        root: { display: 'initial' },
```

Comment:
This was causing the "resize squares" to not display initally when the pool was dragged onto the grid. Changing it to `'initial'` fixes this.